.github/workflows/sandbox.yml

```
@@ -19,14 +19,21 @@ jobs:
         with:
-          go-version: '^1.25'
+          go-version: "^1.26"
 
       - name: Run tests
         run: |
           set -o pipefail
-          go test ./... | tee /tmp/go-test-output.txt
+          go test -coverprofile=coverage.out ./... | tee /tmp/go-test-output.txt
+
+      - name: Upload coverage reports to Datadog
+        if: ${{ secrets.DATADOG_API_KEY != '' && github.actor != 'dependabot[bot]' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)